public/app/core/components/UserEdit/UserProfile.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ChangeEvent, PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { OrgUser, StoreState } from 'app/types';
+import { Input, Button } from '@grafana/ui';
+import { setUserName, setUserEmail, setUserLogin, updateUser } from './state/actions';
+
+export interface Props {
+  adminMode: boolean;
+  uid: number;
+  user: OrgUser;
+  setUserName: typeof setUserName;
+  setUserEmail: typeof setUserEmail;
+  setUserLogin: typeof setUserLogin;
+  updateUser: typeof updateUser;
+}
+
+export interface State {
+  name: string;
+  email: string;
+  login: string;
+}
+
+export class UserProfile extends PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    const { user } = props;
+    this.state = {
+      name: user.name,
+      email: user.email,
+      login: user.login,
+    };
+  }
+
+  onNameChange = name => {
+    this.setState({ name });
+  };
+
+  onEmailChange = email => {
+    this.setState({ email });
+  };
+
+  onLoginChange = login => {
+    this.setState({ login });
+  };
+
+  onUpdateUser = () => {
+    const { uid } = this.props;
+    const { name, email, login } = this.state;
+    this.props.setUserName(name);
+    this.props.setUserEmail(email);
+    this.props.setUserLogin(login);
+    this.props.updateUser(uid);
```

Comment:
is not the updateUser enough here? It calls loadUser on success

Reply:
I definitely can- will just need to pass in those vals. Was just trying to convert to give Permissions bit access to the orgUser. Thanks!